assets/_layout.scss

```diff
@@ -1,24 +1,24 @@
 .page-wrap {
-  padding-bottom: 1em;
-  padding-top: 1em;
-  @media (min-width: 48em) {
-    padding-bottom: 4em;
-    padding-top: 4em;
+  padding-bottom: 1rem;
+  padding-top: 1rem;
+  @media (min-width: 48rem) {
+    padding-bottom: 4rem;
+    padding-top: 4rem;
   }
 }
 .container {
   margin: 0 auto;
-  max-width: 1280px;
-  padding: 0 1em;
+  max-width: 80rem;
+  padding: 0 1rem;
 }
 
 .subpage {
   background: #fff;
-  border: 1px solid #dcdfe6;
-  margin: 1em 0;
-  padding: 1em;
-  @media (min-width: 48em) {
-    margin: 2.5em 0;
-   padding: 2em;
-}
+  border: 0.06rem  solid #dcdfe6;
```

Comment:
Same here, set it to 1px.

Reply:
You mean to set border back to 1px instead of 0.06rem?